### ISSUE TITLE
How to create a palette?

### ISSUE THREAD

**Issue by sommerper** [Donitzo/godot-color-dither#1]
Hi!

I wanted to try this out but got stuck at
```
assign a palette_image containing the original colors in the inspector
```

Not sure what to do here?
Can I use a png from lospec or is a textfile?

Thanks!

**Comment by Donitzo** [comment]
Hello!

You have to use the dither palette generator editor tool script to first produce the dithering palette from a simple image with colors, such as the ones you can download from lospec. You can select how many colors to mix together in the dither.

**Comment by sommerper** [comment]
Thanks for taking the time to reply!

I got as far as you said but I'm unable to drag and drop the image to the `palette image` field. Here I'm using a 1px png from lospec:
![Oct-27-2024 15-17-35](https://github.com/user-attachments/assets/19a72a64-61df-4271-912b-676eef135b45)

I also tried writing the absolute path in the `path` field:
<img width="1599" alt="image" src="https://github.com/user-attachments/assets/efac504a-4f07-40a5-899a-c317fb653df4">

Maybe I just don't understand the Image type - or maybe it's because I'm using Godot 4.4dev3.
